modules/ppcp-onboarding/src/OnboardingRestController.php

```diff
@@ -16,7 +16,7 @@
 /**
  * Exposes and handles REST routes related to onboarding.
  */
-class Onboarding_REST_Controller {
+class OnboardingRestController {
```

Comment:
Similar, to the above, REST is an acronym.

Reply:
✔️ 9f2496a